### PR TITLE
Allow injecting the service into Controller constructor for autowiring

### PR DESCRIPTION
**Pull request by andrewmy**
This update allows injecting the service into Controller constructor for autowiring.

### CODE REVIEW

File: Resources/config/breadcrumbs.xml [hunk 1]
```diff
@@ -6,11 +6,13 @@
     <services>
 
         <!-- Our service, for controllers -->
-        <service id="white_october_breadcrumbs" class="WhiteOctober\BreadcrumbsBundle\Model\Breadcrumbs" public="true">
+        <service id="WhiteOctober\BreadcrumbsBundle\Model\Breadcrumbs" class="WhiteOctober\BreadcrumbsBundle\Model\Breadcrumbs" public="true">
```
Comment (bocharsky-bw) [inline Resources/config/breadcrumbs.xml]: Should be private from now

File: Resources/config/breadcrumbs.xml [hunk 2]
```diff
@@ -6,11 +6,13 @@
...
             <call method="setRouter">
                 <argument type="service" id="router" />
             </call>
         </service>
+        <service id="white_october_breadcrumbs" alias="WhiteOctober\BreadcrumbsBundle\Model\Breadcrumbs">
```
Comment (bocharsky-bw) [inline Resources/config/breadcrumbs.xml]: should remain public

File: Resources/config/breadcrumbs.xml (hunk 1 above)
Comment (andrewmy) [reply Resources/config/breadcrumbs.xml]: Makes sense, thanks

**Comment by andrewmy** [comment]
Ready for merge?

**Comment by andrewmy** [comment]
Cool! Can't wait for a point release ;)

**Comment by bocharsky-bw** [comment]
For a possible workaround, until it's merged, you can add an alias in your system so you can use it for now:
```
# config/services.yaml
services:
    # ...
    WhiteOctober\BreadcrumbsBundle\Model\Breadcrumbs: '@white_october_breadcrumbs'
```

**Comment by sampart** [comment]
Thank you for this, @andrewmy.  Sorry for the delay in merging.

Thank you @bocharsky-bw for reviewing and approving as well - makes my job a lot easier!

**Comment by sampart** [comment]
This is now in release https://github.com/whiteoctober/BreadcrumbsBundle/releases/tag/1.4.2

**Pull request merged**